src/AutoPlay.ts

```diff
@@ -42,10 +47,12 @@ class AutoPlay implements Plugin {
    */
   public constructor({
     duration = 2000,
+    animationDuration = undefined,
```

Comment:
Changing the order of parameters of constructors and methods is dangerous.

Use it for the last parameter.

Reply:
Thanks for checking dangerous point!

I have a question about this. `AutoPlayOptions` object is received as an parameters to the constructor, is it dangerous even if the order of keys inside the object is changed?

Reply:
@malangfox 

oh sorry i was wrong